### DISCUSSION TITLE
How can I set up my custom theme on React Spectrum?

### DISCUSSION THREAD

**Discussion by elVengador** [adobe/react-spectrum#5779] [Q&A] [answered]
Hello everyone, I would like to set up my `custom theme` on react spectrum to use on my personal page, how can I do it?
I understand how to set the theme, but I don't know how to build a new theme
```
<Provider theme={defaultTheme}>
```

Also, I printed the `defaultTheme` variable, but I don't understand that values
```
{
    "global": {
        "spectrum": "zA6MfG_spectrum",
        "spectrum--dark": "zA6MfG_spectrum--dark",
        "spectrum--darkest": "zA6MfG_spectrum--darkest",
        "spectrum--large": "zA6MfG_spectrum--large",
        "spectrum--light": "zA6MfG_spectrum--light",
        "spectrum--lightest": "zA6MfG_spectrum--lightest",
        "spectrum--medium": "zA6MfG_spectrum--medium"
    },
    "light": {
        "spectrum--light": "PFjRbG_spectrum--light"
    },
    "dark": {
        "spectrum--darkest": "gP7gvq_spectrum--darkest"
    },
    "medium": {
        "spectrum--medium": "xSyFOq_spectrum--medium"
    },
    "large": {
        "spectrum--large": "HAZavG_spectrum--large"
    }
}
```

**Comment by reidbarber** [comment, marked as answer]
Unfortunately, customizing the React Spectrum theme isn't officially supported. You can read more about why this is the case in https://github.com/adobe/react-spectrum/issues/3370 and https://github.com/adobe/react-spectrum/issues/3624.

**Reply by elVengador** [reply]
Got it, thanks for the information
